watchtower/wtclient/backup_task.go

```diff
@@ -192,13 +192,25 @@ func (t *backupTask) bindSession(session *wtdb.ClientSessionBody) error {
 		}
 	}
 
-	// All justice transactions have a p2wkh output paying to the victim.
-	weightEstimate.AddP2WKHOutput()
+	// All justice transactions will either use segwit v0 (p2wkh + p2wsh)
+	// or segwit v1 (p2tr).
+	switch len(t.sweepPkScript) {
```

Comment:
nit: couldn't we just parse it

Reply:
You mean like fully parse it to get the proper concrete type? FWIW, this is the client itself, and at this point we always make p2tr addresses, so we have confidence here we're not dealing with some exotic address type.

Reply:
Ended up parsing it.